### ISSUE TITLE
What is the meaning of this error? 

### ISSUE THREAD

**Issue by clementcrownrise** [josegonzalez/cakephp-upload#422]
![image](https://cloud.githubusercontent.com/assets/3583726/21030417/9e22d8e0-bd9e-11e6-9260-3e7aa0bb845d.png)

I got this error after clicking on the "submit " button of my form.

Please what is the meaning, how can I get it fixed?

Thanks

**Comment by clementcrownrise** [comment]
I used composer, and I also downloaded manually.

**Comment by josegonzalez** [comment]
Do not download it manually. In fact, remove the manually downloaded code, then re-run via composer.

Is there a reason you manually downloaded it?

**Comment by clementcrownrise** [comment]
I downloaded it manually because it was giving error after installing with composer, anyway, let me use composer again.

**Comment by josegonzalez** [comment]
What was the error on composer install?

**Comment by clementcrownrise** [comment]
Let me try again.

**Comment by josegonzalez** [comment]
Can you do the following:

```
rm -rf vendor
composer install
```

**Comment by clementcrownrise** [comment]
I am using windows, it is complaining
![image](https://cloud.githubusercontent.com/assets/3583726/21031095/0a373ac8-bda2-11e6-86c8-7857497a24ee.png)

**Comment by josegonzalez** [comment]
Just delete that vendor directory in `Windows Explorer` and then run `composer install`.

**Comment by clementcrownrise** [comment]
You mean I should install the plugin again or just composer?

**Comment by clementcrownrise** [comment]
I just did that , it still shows "Nothing to install"

![image](https://cloud.githubusercontent.com/assets/3583726/21031935/8546a09c-bda6-11e6-90c7-9f0b107200f5.png)

**Comment by josegonzalez** [comment]
Can you hop on the cakephp slack or irc room? I am `savant`on both.

**Comment by clementcrownrise** [comment]
Ok, i am logged in to IRC Room.

Thanks

**Comment by xhs345** [comment]
I see you always try to run composer from the /bin directory. Try it from the /billboard directory

**Comment by clementcrownrise** [comment]
![image](https://cloud.githubusercontent.com/assets/3583726/21072030/d034072a-beb4-11e6-98b8-745d4b240b7a.png)


This is the updated screenshot for installation.

**Comment by xhs345** [comment]
Ok, looks good. So is it working now or do you still get an error?

**Comment by clementcrownrise** [comment]
@josegonzalez  ,  @xhs345 , Thanks so much it is working now, @josegonzalez  but I would like if you could help me cos I need little modification on the directory of the upload.

I do not want folders to be created
